gene2phenotype_project/gene2phenotype_app/serializers/user.py

```diff
@@ -72,6 +72,34 @@ def panels_names(self, id):
 
         return user_panels
 
+    def check_panel_permission(self, panel_name):
+        """
+            Check if user has permission to edit the panel.
+
+            Args:
+                    self: user
+                    panel_name: name of the panel
+
+            Returns:
+                        1 if user has permission to edit
+                        0 if user does not have permission to edit
```

Comment:
can the function return true or false instead of 0 or 1 since that will be more clear?

Reply:
I found a bug in this method.
I updated the code + the return type.